agent/agent-tooling/src/test/resources/applicationinsights_with_utf8_bom.json

```diff
@@ -0,0 +1,3 @@
+﻿{
+  "connectionString": "InstrumentationKey=00000000-0000-0000-0000-000000000000"
+}
```

Comment:
even though it doesn't show up here, I verified locally after deleting, and pulling back from git, that this file has the byte order marker (BOM) at the beginning, you should be able to see it in Intellij:

![image](https://user-images.githubusercontent.com/218610/188009942-75a8ffab-66a2-4ac1-8a06-d3afb94f63ac.png)